themes/mms-cloud/theme.conf

```diff
@@ -12,4 +12,7 @@ version_selector = VERSION_SELECTOR
 stable = STABLE
 has_mdb_agent =
 edition = EDITION
-nav_excluded = NAV
\ No newline at end of file
+nav_excluded = NAV
+automation_eol =
```

Comment:
Just want to confirm this is not supposed to have a value?

Reply:
Yes, the value is specified in mms-docs.